python/triton/compiler.py

```diff
@@ -19,6 +19,7 @@
 from typing import Any, Callable, Dict, Optional, Tuple, Type, Union
 
 import setuptools
+import torch
```

Comment:
Please don't add back torch as a core dependency

Reply:
Sure. I don't know how this line kicked in. It wasn't my intention at all.